microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -970,6 +970,33 @@ WebView2 provides functionality to handle the JavaScript function `window.open()
 
 ---
 
+
+#### todo: add link to the above tab-set?  Nov 2023 incoming
```

Comment:
I don't think we need to add to this one. It's a property of a class that's already existing

Reply:
removed temp section